chrome/content/zotero/xpcom/itemPaneManager.js

```diff
@@ -0,0 +1,358 @@
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2024 Corporation for Digital Scholarship
+					 Vienna, Virginia, USA
+					 https://digitalscholar.org
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero.  If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+/**
+ * @typedef SectionIcon
+ * @type {object}
+ * @property {string} icon - Icon URI
+ * @property {string} [darkIcon] - Icon URI in dark mode. If not set, use `icon`
+ * @typedef SectionL10n
+ * @type {object}
+ * @property {string} l10nID - data-l10n-id for localization
+ * @property {string} [l10nArgs] - data-l10n-args for localization
+ * @typedef SectionButton
+ * @type {object}
+ * @property {string} type - Button type, must be valid DOMString and without ","
+ * @property {(options: SectionEventHookArgs) => void} onClick - Button click callback
+ * @typedef SectionBasicHookArgs
+ * @type {object}
+ * @property {string} paneID - Registered pane id
+ * @property {Document} doc - Document of section
+ * @property {HTMLDivElement} body - Section body
+ * @typedef SectionUIHookArgs
+ * @type {object}
+ * @property {() => {item: Zotero.Item, mode: string, inTrash: boolean, tabType: string}} getData - Get section data
+ * @property {(l10nArgs: string) => void} setL10nArgs - Set l10n args for section header
+ * @property {(l10nArgs: string) => void} setEnabled - Set pane enabled state
+ * @property {(summary: string) => void} setSectionSummary - Set pane section summary
+ * @property {(buttonType: string, options: {disabled?: boolean; hidden?: boolean}) => void} setSectionButtonStatus - Set pane section button status
```

Comment:
Added new methods to hook callback to allow plugins to set summary or change section header button status.